lib/restaurant.rb

```diff
@@ -0,0 +1,17 @@
+class Restaurant
+
+  attr_reader :menu
+
+  def initialize(menu)
+    @menu = menu
+  end
+
+  def submit_order(customer_name, customer_phone_number, order_dishes)
+    order_dishes.each do
+      |dish| fail "Dish not on menu" if !@menu.dishes.include? dish
```

Comment:
Style/MultilineBlockLayout: Block argument expression is not on the same line as the block start.<br>Style/NegatedIf: Favor `unless` over `if` for negative conditions.